engine/components/physics/box2d/IgeEntityPhysics.js

```diff
@@ -77,8 +77,6 @@ var IgeEntityPhysics = IgeEntity.extend({
 		var filterCategoryBits = 0x0002;
 		if (this._category === 'units') {
 			filterCategoryBits = 0x0002;
-		} else if (this._category === 'debris') {
```

Comment:
I think this will affect physics. If we no longer include these bits in the bitmask, collision behaviour will change.

Reply:
ok did a check, and it appears that we're checking collisions using bitwise `&` so removing this bit should have no effect at all

Reply:
it looks like this goes for all the box2d engines we use as well